tests/test_tasklist.py

```diff
@@ -76,3 +76,7 @@ def test_tasklist_for_submit_application_route(
     assert b"Application complete" in response.data
     assert b"Return to your applications page" in response.data
     assert b"test@example.com" in response.data
+
+
+def test_get_blank_sections():
+    
```

Comment:
This is just an empty function, are we planning on adding a test here?

Reply:
Sorry forgot to remove this test. No need for test in here as we moved the static function to class function & that class is being tested.

Reply:
I think this is an old push that didn't get refreshed by Github? 🤔